bin/monit-dashboard.py

```diff
@@ -64,35 +66,45 @@ def getMonit():
                 checks[name] = status[name]
 
             sorted_checks = OrderedDict()
-            sorted_checks = OrderedDict(sorted(checks.iteritems(), key=itemgetter(1), reverse=True))
+            sorted_checks = OrderedDict(sorted(checks.iteritems(),
+                key=itemgetter(1), reverse=True))
             count = calculate_count(sorted_checks)
-            server = dict(name=site, url=s['url'], result=sorted_checks, s_rate=count)
+            server = dict(name=site, url=s['url'],
+                result=sorted_checks, s_rate=count)
 
             output.append(server)
-
     print(datetime.datetime.now())
     return(output)
 
 # Classes
 
 class monitDashboard(web.application):
-
     def run(self, port=8080, *middleware):
         func = self.wsgifunc(*middleware)
         return web.httpserver.runsimple(func, ('0.0.0.0', port))
 
 
 class index(object):
-
     def GET(self):
-        return render.index(output=getMonit(), now=datetime.datetime.now())
-
+        return render.index(output=getMonit(),
+            now=datetime.datetime.now(),
+            download_api=os.environ['DOWNLOAD_API_URL'])
 
 class help(object):
-
     def GET(self):
         return render.help()
 
+class download(object):
+    def GET(self):
+        filename = 'health_report.xlsx'
```

Comment:
That should be added in the `.gitignore`

Reply:
done